docs/src/pages/components/text/examples/GlobalStyling.tsx

```diff
@@ -0,0 +1,7 @@
+import { Text } from '@aws-amplify/ui-react';
+
+export const GlobalStylingExample = () => (
+  <Text className="primary-styled-variable-text" variation="primary">
+    {'This is my styled text'}
```

Comment:
nit: do we need the {' '} around "This is my styled text"

Reply:
Nope. I copied and pasted the code and didn't pay attention 🤷

Reply:
Removed